apps/shared/range_parameter_controller.cpp

```diff
@@ -70,9 +71,14 @@ bool RangeParameterController::textFieldDidFinishEditing(TextField * textField,
   return false;
 }
 
+void RangeParameterController::setRange(InteractiveCurveViewRange * range){
+  m_interactiveRange = range;
+  m_tempInteractiveRange = *range;
```

Comment:
Same could be achieved here to avoid keeping track of useless InteractiveCurveViewRange members.